composite/market-closure/src/checks/schedule.ts

```diff
@@ -0,0 +1,13 @@
+import { ScheduleCheck } from './index'
+import { MarketClosure, Schedule } from 'market-closure'
+
+export const scheduleExecute: ScheduleCheck = async (schedule: Schedule) => {
```

Comment:
No need for `async` as we are not awaiting anything in this fn, but it would be great to declare a return type:
```suggestion
export const scheduleExecute: ScheduleCheck = (schedule: Schedule): Promise<?> => {
```

Reply:
Since `ScheduleCheck` is already a defined type, is it necessary to also define the types here?

Just wondering what's best practice in TS

Reply:
Aha, yes! Sorry about that, I missed it.

As we already declare the var as `ScheduleCheck` there is no need to add return types. I think even parameter types are unnecessary at this point.